scripts/dockerNodeStats/launch.bash

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+#
+
+set -o nounset
+set -o pipefail
+IFS=$'\n\t'
+
+if [ -f ".venv/bin/activate" ] ; then
+	sleep 0
+else
+    echo "Installing and creating venv.."
+    pip3 install virtualenv >/dev/null 2>&1
+    virtualenv -p python3 .venv >/dev/null 2>&1
+fi
+set -o allexport
```

Comment:
question: why don't you run these things in a docker container? then you would not contaminate the machines where you run this. There are plenty of examples in osparc-simcore/scripts

Reply:
I agree.

Reply:
done